core/GlobalState.cc

```diff
@@ -1743,21 +1934,29 @@ unique_ptr<GlobalStateHash> GlobalState::hash() const {
     u4 hierarchyHash = 0;
     UnorderedMap<NameHash, u4> methodHashes;
     int counter = 0;
-    for (const auto &sym : this->symbols) {
-        if (!sym.ignoreInHashing(*this)) {
-            if (sym.isMethod()) {
-                auto &target = methodHashes[NameHash(*this, sym.name.data(*this))];
-                target = mix(target, sym.hash(*this));
-                hierarchyHash = mix(hierarchyHash, sym.methodShapeHash(*this));
-            } else {
-                hierarchyHash = mix(hierarchyHash, sym.hash(*this));
+
+    for (const auto *symbolType : {&this->classAndModules, &this->fields, &this->typeArguments, &this->typeMembers}) {
+        counter = 0;
+        for (const auto &sym : *symbolType) {
+            hierarchyHash = mix(hierarchyHash, sym.hash(*this));
+            counter++;
+            if (DEBUG_HASHING_TAIL && counter > symbolType->size() - 15) {
+                errorQueue->logger.info("Hashing symbols: {}, {}", hierarchyHash, sym.name.show(*this));
```

Comment:
```suggestion
                errorQueue->logger.info("Hashing class symbols: {}, {}", hierarchyHash, sym.name.show(*this));
```

Reply:
I was wrong yesterday; this line processes class, fields, type arguments, and type members.